pipelines/openjdk10_nightly_pipeline.groovy

```diff
@@ -1,14 +1,14 @@
 println "building ${JDK_VERSION}"
 
-def buildPlatforms = ['Mac', 'Windows', 'Linux', 'zLinux', 'ppc64le', 'AIX', 'arm64']
+def buildPlatforms = ['Mac', 'Windows', 'Linux', 'zLinux', 'ppc64le', 'AIX', 'aarch64']
```

Comment:
This value is used as index. Either arm64 or aarch64 is ok.

Reply:
I think I will change it anyway (even if change not needed, reads better).